Makefile

```diff
@@ -292,13 +294,28 @@ manifests: controller-gen yq/install kcp-plugin
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..."
 	${YQ} e '.metadata.name = "compute-operator-manager-role"' config/rbac/role.yaml > deploy/compute-operator/clusterrole.yaml && \
 	${YQ} e '.metadata.name = "leader-election-operator-role" | .metadata.namespace = "{{ .Namespace }}"' config/rbac/leader_election_role.yaml > deploy/compute-operator/leader_election_role.yaml && \
-	kubectl kcp crd snapshot --filename config/crd/singapore.open-cluster-management.io_registeredclusters.yaml --prefix today-`date +%Y-%m-%d` \
+	kubectl kcp crd snapshot --filename config/crd/singapore.open-cluster-management.io_registeredclusters.yaml --prefix latest \
```

Comment:
ci/prow/check is failing on a `kubectl kcp` call.  Might need to ensure in Makefile that kcp plugin is installed

Reply:
There is a target in there for that but must not be working. I'm going to go ahead and merge this one and we can investigate that issue separately since it already exists in main. Thanks for the heads up.